lua/tman.lua

```diff
@@ -194,4 +214,51 @@ M.goToTerm = function ()
     end
 end
 
+---@mod tman.sendCommand Sending Commands
+---@tag tman.sendCommand
+---@param cmd string command to be sent to the terminal
+---       end with '\r' to send a carriage return
+---@param opts table optional configuration options:
+---  * {split} (string) optional "right" or "bottom"
+---    defaults to the value set at the time of setup
+---  * {pre} (string) optional command to run before sending the command
+---  * {scrollTop} (boolean} optional scroll the terminal prompt to top before executing command
+---                 defaults to true
+M.sendCommand = function(cmd, opt)
+    if rawget (opt, "scrollTop") == nil then
+        opt.scrollTop = true
+    end
+
+    opt = M.validateAndGetWithSplit(opt)
+    local oldTbl = {
+        split = tman.split,
+        width = tman.width,
+        height = tman.height,
+    }
+    local tbl = {
+        split = opt.split or oldTbl.split,
+        width = opt.width or oldTbl.width,
+        height = opt.height or oldTbl.height,
+    }
+    M.setup(tbl)
```

Comment:
Check if the window options are still kept after running some command (if `wo` passed at setup is different from default values)